client/src/components/Room.js

```diff
@@ -16,13 +16,13 @@ function Room(): React$Element<any> {
 		else toast.info('Share the link with your friends!');
 	}, [history, id, user]);
 	
-	return (
+	return user ? (
```

Comment:
this fix a minor bug, if there is no user, we do not need to render the room because this will cause petitions to the backend with the lobby